src/Lecture2.hs

```diff
@@ -194,7 +268,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] list2 = list2
+merge list1 [] = list1
+merge (x1:list1) (x2:list2)
+  | x1 < x2 =   x1 : merge list1 (x2:list2)
+  | otherwise = x2 : merge (x1:list1) list2
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️